lansa-vmss-windows-autoscale-sql-database/azuredeploy.parameters.json

```diff
@@ -2,17 +2,62 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "vmssName": {
+        "stackName": {
             "value": "GEN-UNIQUE-10"
         },
+        "certificateBase64Encoded": {
+            "value" : "GEN-SELFSIGNED-CERT-PFXDATA"
+        },
+        "certificatePassword": {
+            "value": "GEN-SELFSIGNED-CERT-PASSWORD"
+        },
+        "databaseNewOrExisting": {
+            "value": "new"
+        },
+        "databaseServerName": {
+            "value": "GEN-UNIQUE-10"
+        },
+        "databaseName": {
+            "value": "lansa"
+        },
+        "databaseLogin": {
+            "value": "GEN-UNIQUE-10"
+        },
+        "databaseLoginPassword": {
+            "value": "GEN-PASSWORD"
+        },
+        "adminUsername": {
+            "value": "GEN_UNIQUE-10"
+        },
         "adminPassword": {
             "value": "GEN-PASSWORD"
         },
+        "webUsername": {
+            "value": "GEN_UNIQUE-10"
+        },
         "webPassword": {
             "value": "GEN-PASSWORD"
         },
-        "databaseLoginPassword": {
-            "value": "GEN-PASSWORD"
+        "webServerMaxConnect": {
```

Comment:
remove all of these and use defaultValues in the template (essentially, remove all your non-GEN parameter values)

Reply:
Done